server/server.go

```diff
@@ -1415,6 +1415,10 @@ func (s *Server) v1DataGet(w http.ResponseWriter, r *http.Request) {
 		DecisionID: decisionID,
 	}
 
+	if input == nil {
```

Comment:
I think this validates if the 'input' http query parameter is present in the request, the validation should be after JSON  unmarshall is completed  and validate if it has an 'input' field.

Reply:
That's a good point, Christian! The tests should help shine some light on this :)

Reply:
Making the server generate a warning on GET w/o an input document feels a bit wrong to me. It's not invalid to execute a policy query without supplying an input document. The POST endpoint is specifically for executing policies with an input document provided--so let's warn if users invoke that without supplying input--but for GET, I'll be a bit sad if running `curl localhost:8181/v1/data` complains at me 😿